### PR TITLE
SLING-7454: Make the IT use the latest repoinit snapshots, enable dis…

### PR DESCRIPTION
**Pull request by karlpauls**
…abled IT's that require the newer snapshots, and improve backwardscompatibilty by dependening on older version.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -110,35 +110,40 @@
                 <!-- the Sling instance is provisioned from the model in src/main/provisioning/model.txt -->
                 <groupId>org.apache.sling</groupId>
                 <artifactId>slingstart-maven-plugin</artifactId>
-                <version>1.7.0</version>
```
Comment (kwin) [inline pom.xml]: This version is usually managed. 1.7.16 should be managed in the latest parent pom.

File: pom.xml [hunk 2]
```diff
@@ -110,35 +110,40 @@
...
                         <goals>
                             <goal>prepare-package</goal>
                         </goals>
                     </execution>
                     <execution>
                         <id>build-launchpad-package</id>
+                        <phase>pre-integration-test</phase>
                         <goals>
                             <goal>package</goal>
                         </goals>
```
Comment (kwin) [inline pom.xml]: You should configure attachArtifact to false to prevent the IT launchpad from being deployed with the release.

File: pom.xml [hunk 3]
```diff
@@ -110,35 +110,40 @@
...
                     </execution>
                     <execution>
                          <id>start-container-before-IT</id>
+                        <phase>pre-integration-test</phase>
```
Comment (kwin) [inline pom.xml]: this is the default phase for start

File: pom.xml [hunk 4]
```diff
@@ -110,35 +110,40 @@
...
                         <goals>
                             <goal>start</goal>
                         </goals>
                     </execution>
                     <execution>
                          <id>stop-container-after-IT</id>
+                         <phase>post-integration-test</phase>
```
Comment (kwin) [inline pom.xml]: this is the default phase for stop

File: pom.xml (hunk 1 above)
Comment (karlpauls) [reply pom.xml]: I agree that usually is the case. It is not in this case. In parent 30 it is 1.7.2 and this needs 1.7.16 - updating to the latest parent has other problems which I don't feel like addressing right now.

File: pom.xml (hunk 2 above)
Comment (karlpauls) [reply pom.xml]: Right, that makes sense.

File: pom.xml (hunk 3 above)
Comment (karlpauls) [reply pom.xml]: Yeah, I added that mostly for consistency as I need the other executions moved - but if you think it is clear enough without that I'm fine with removing it.

File: pom.xml (hunk 4 above)
Comment (karlpauls) [reply pom.xml]: Yeah, I added that mostly for consistency as I need the other executions moved - but if you think it is clear enough without that I'm fine with removing it.

**Comment by karlpauls** [comment]
@kwin, I updated the PR to not attach artifacts and remove the unnecessary phases - thanks a lot for looking at it!

**Pull request merged**